template/config/vitest/src/components/__tests__/HelloWorld.spec.js

```diff
@@ -5,6 +5,8 @@ import HelloWorld from '../HelloWorld.vue'
 
 describe('HelloWorld', () => {
   it('renders properly', () => {
+    // TODO: https://github.com/vuejs/vue-test-utils/issues/2087
+    // @ts-ignore
```

Comment:
I don't think we should merge like this, so we'll have to wait for the fix (but it may take a long time as VTU v1 is not really active).

Reply:
First of all, you're right.
Because there are still lists left to complete.

For that matter, the problem actually existed before I modified this project.
I'm also wondering how he (Haoqun Jiang) avoided this problem, after all, the last version already depends on Vue 2.7.x already!

How about we ask "Haoqun Jiang" for advice?

Reply:
TBH I have no idea how to fix that type error…

Reply:
A workaround was proposed in the issue I opened previously: https://github.com/vuejs/vue-test-utils/issues/2026
But I'm no expert on TypeScript, so I don't know if that's the right fix.

Reply:
TBH. 
It would be best to have [vue-test-utils](https://github.com/vuejs/vue-test-utils) aligned with "Vue 2.7.16".

But that repository is not very active.

I've also forked [vue-test-utils](https://github.com/vuejs/vue-test-utils) aligned to try to fix this, which is a bit of a challenge. 

A more practical solution for now is to create a corresponding Typescript shim in the output template，like https://github.com/vuejs/vue-test-utils/issues/2026#issuecomment-1460894573

Reply:
@cexbrayat Hi! 
Excuse me!
Could you please review the latest submission and provide feedback on whether it is an acceptable solution?

https://github.com/vuejs/create-vue/pull/413/commits/b1bae5e6c74e1b1c448eb8491d291f85217bdf04

Reply:
It's fine by me. Obviously it would be better to open a PR on VTU to fix it properly and get a new release, but I'm not sure many people will generate a new Vue 2 project with create-vue, so that's probably OK.

Reply:
> Obviously it would be better to open a PR on VTU to fix it properly and get a new 
release,

Since the VTU author is currently focusing on "Vue 3", I can follow up in the future when this issue is fixed.

Possibly (https://github.com/vuejs/vue-test-utils/issues/2087#issuecomment-1920627565) I'll open a PR 

For now. I think this solution is sufficient